imports/ui_3/views/components/chatbox.html

```diff
@@ -53,7 +53,7 @@
                     <button class="btn btn-white btn-xs js-like">
                         <i class="fa {{#if this.isLikedBy currentUser._id}} fa-exclamation-circle {{else}} fa-hand-o-up {{/if}}"></i>
                         {{_ 'important'}}!
-                        ({{likesCount}})
+                        ({{likesCount this}})
```

Comment:
a this.likesCount nem oldja meg?
Elegansabb lenne, mint elvarni, hogy beadja valaki a kontextust.
(A helperek pont azert izlesesesebbek, mert nem kell beadni mindig a kontextust)